src/plugins/home/server/tutorials/haproxy_metrics/index.ts

```diff
@@ -1,76 +0,0 @@
-/*
```

Comment:
was this file previously this created by you?

Reply:
Nope, I will add it back